workflows/n_t.snake

```diff
@@ -283,46 +257,54 @@ rule smcpp_plot:
     input: 
         rules.run_smcpp.output
     output: 
-        output_dir + "/Results/{seeds}/trees.smc.gz.final.json.csv"
+        output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/trees.smc.gz.final.json.csv"
     run:
         # TODO get the genetion time from std source
-        smc.run_smcpp_plot(input[0], output[0], generation_time=generation_time)
-
-
-def ne_files_smcpp(wildcards):
-    return expand(output_dir + "/Results/{seeds}/trees.smc.gz.final.json.csv",
-            seeds=seed_array, chrms=chrm_list)
+        smc.run_smcpp_plot(input[0], output[0], generation_time=generation_time_helper(wildcards,species))
 
 
 rule compound_smcpp:
-    input: ne_files_smcpp
-    output: output_dir + "/Results/smcpp_estimated_Ne.png"
-    run: plots.plot_compound_smcpp(input, output[0], model, num_sampled_genomes_per_replicate, generation_time)
+    input: output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/trees.smc.gz.final.json.csv"
+    output: output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/smcpp_estimated_Ne.png"
+    run: 
+        model = species.get_demographic_model(wildcards.demog)
+        plots.plot_compound_smcpp(input, output[0], model, num_sampled_genomes_per_replicate, 
+        generation_time_helper(wildcards,species),
+        )
 
 
 # ###############################################################################
 # MSMC
 # ###############################################################################
 
 rule ts_to_multihep:
-    input: rules.simulation.output
-    output: output_dir + "/Intermediate/{seeds}/{samps}.{chrms}_{selection}.trees.multihep.txt"
-    run: msmc.write_msmc_file(input[0], num_sampled_genomes_msmc, selection, mask_file)
+    input: output_dir + "/simulated_data/{demog}/{dfes}/{annots}/{seeds}/sim_{chrms}.trees"
+    output: output_dir + "/inference/smc/{demog}/{dfes}/{annots}/{seeds}/{samps}.{chrms}.trees.multihep.txt"
+    run: 
+        print(input[0], num_sampled_genomes_msmc, mask_file)
+        msmc.write_msmc_file(input[0], output[0], num_sampled_genomes_msmc, mask_file)
 
 
 rule run_msmc:
     input:
         # TODO make this NOT dependent on all simulations.
         # is there anyway to get access to wildcards.seeds from here? 
-        expand(output_dir + "/Intermediate/{seeds}/{samps}.{chrms}_{selection}.trees.multihep.txt",
-            chrms=chrm_list, seeds=seed_array, samps=num_sampled_genomes_msmc, selection=selection)
-    output: output_dir + "/Intermediate/{seeds}/{samps}.{selection}.trees.multihep.txt.final.txt"
+        #expand(output_dir + "/Intermediate/{seeds}/{samps}.{chrms}_{selection}.trees.multihep.txt",
```

Comment:
I think you can delete if is the same thing as above. I see that above we were previously asking for the mask_file name. Can you include this into the output_directories somehow? Thanks